### PR TITLE
Add Contact us page

### LINKED ISSUE #93: Add Contact us page
**User story**
As a user, I want to be able to contact the website owner so that I can contact them when I have a question. 

**Acceptance criteria**
* Create ContactUS page that has the introduction of team Daebak and team members' name, GitHub and LinkedIn
* Connect with navbar menu

### LINKED ISSUE #110: [Improvement] Update App name
**User story**
As a user/developer, I want to be able to see app name on the top 

**Acceptance criteria**
* Add Think Small title on the app 
image

### PR DESCRIPTION
**Pull request by anhyesun**
#### [Add Contact us page](#)

**Checklist**
- [x] Run `npm run lint:fix` or `yarn lint:fix`
 
**What and how**
Responsive is not well working yet (need help!)

**Screenshots**
contact

### CODE REVIEW

File: client/src/components/Form/RequestForm.js [hunk 1]
```diff
@@ -65,8 +65,8 @@ const Form = ({
                 <div className={classes.tags}>
                     {tags.map(({ label, selected }, i) => (
                         selected
-                        ? <SelectedChip label={label} onClick={() => selectTag(i)} />
-                        : <UnselectedChip label={label} onClick={() => selectTag(i)} />
+                            ? <SelectedChip label={label} onClick={() => selectTag(i)} />
```
Comment (yehee) [inline client/src/components/Form/RequestForm.js]: missing `key` attribute

File: client/src/modules/about.module.css [hunk 2]
```diff
@@ -17,8 +17,8 @@
     margin: auto;
     display: block;
     object-fit: cover;
-    height: 230px;
-    width: 230px;
+    height: 200px;
```
Comment (yehee) [inline client/src/modules/about.module.css]: what's the reason behind these changes?

File: client/src/modules/contact.module.css [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    width: 180px;
+}
+
+.square {
+    border-radius: 7.5px;
+    display: flex;
+    flex-direction: column-reverse;
+    height: 300px;
+    margin: 20px auto;
+    padding: 10px 10px 10px;
```
Comment (yehee) [inline client/src/modules/contact.module.css]:
```suggestion
    padding: 10px;
```

File: client/src/pages/ContactPage.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+const Square = ({
+    name, github, linkedin, src,
+}) => (
+    <div className={classes.square}>
+        <div>
+            <img src={src} className={classes.img} alt={name} />
+            <h3>{name}</h3>
+            <a href={github}>
+                {' '}
```
Comment (yehee) [inline client/src/pages/ContactPage.js]: could this be done using margin or padding?

File: client/src/pages/ContactPage.js [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+)
+
+const Contact = () => (
+    <div className={classes.root}>
+
+        <div className={classes.title}>
+            <h2> Contact Developers </h2>
+        </div>
+        <div className={classes.subtitle}>
+            {' '}
```
Comment (yehee) [inline client/src/pages/ContactPage.js]: same, could be done by tweaking margin

File: client/src/pages/ContactPage.js [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+                {' '}
+                <LinkedInIcon />
+            </a>
+        </div>
+    </div>
+)
+
+const Contact = () => (
+    <div className={classes.root}>
+
```
Comment (yehee) [inline client/src/pages/ContactPage.js]: no extra newline

File: client/src/pages/ContactPage.js [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+            {' '}
+        </div>
+        <div className={classes.sub}>
+            <p>DaeBak is Korean slang for “awesome”,” cool”, or “great success”. We want to make an awesome community for small businesses that will eventually bring them this great success. </p>
+        </div>
+        <div className={classes.sub}>
+            <p>We are 4th-year UBC students who are looking for an internship and full-time opportunity. Please feel free to reach out to us. </p>
+        </div>
+        <div className={classes.container}>
+            <Square
```
Comment (yehee) [inline client/src/pages/ContactPage.js]: can they be put into an array and have `map` instead?

File: client/src/modules/contact.module.css [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+
+.root {
+    display: inline;
+    height: 100%;
+}
+
+.title {
+    font-family: 'Baloo 2', cursive;
+    text-transform: uppercase;
+    width: 926px;
```
Comment (yehee) [inline client/src/modules/contact.module.css]: these `width: 926px` will be fixed until the screen size hits `768px`, so until then it would be wider than the actual screen size

Comment (yehee) [reply client/src/modules/contact.module.css]: also no margin when the screen size shrinks?
Screen Shot 2020-07-30 at 10 29 33 AM

Comment (yehee) [reply client/src/modules/contact.module.css]: also the width of the `title` and `subtitle` is not responsive

Comment (anhyesun) [reply client/src/modules/contact.module.css]: Done!

File: api/package.json [hunk 9]
```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "start": "REACT_APP_WEBSITE_URL=https://thinksmall.herokuapp.com node ./bin/www",
-    "dev": "nodemon ./bin/www"
+    "dev": "REACT_APP_WEBSITE_URL=http://localhost:8080 nodemon ./bin/www"
```
Comment (yehee) [inline api/package.json]: This shouldn't be included

Comment (anhyesun) [reply api/package.json]: No more here :)

File: client/src/components/Form/RequestForm.js [hunk 10]
```diff
@@ -70,9 +70,15 @@ const Form = ({
                 <Text>Request Details</Text>
                 <div className={classes.tags}>
                     {tags.map(({ label, selected }, i) => (
-                        selected
-                            ? <SelectedChip key={label} label={label} onClick={() => selectTag(i)} />
-                            : <UnselectedChip key={label} label={label} onClick={() => selectTag(i)} />
+                        <div key={i}>
```
Comment (yehee) [inline client/src/components/Form/RequestForm.js]: What happened here? This would print out the logic, not evaluating it?

File: client/src/pages/ContactPage.js [hunk 11]
```diff
@@ -0,0 +1,81 @@
...
+            <h3> Team DaeBak </h3>
+        </div>
+        <div className={classes.sub}>
+            <p>DaeBak is Korean slang for “awesome”,” cool”, or “great success”. We want to make an awesome community for small businesses that will eventually bring them this great success. </p>
+        </div>
+        <div className={classes.sub}>
+            <p>We are 4th-year UBC students who are looking for internship and full-time opportunity. Please feel free to reach out to us. </p>
+        </div>
+        <div className={classes.container}>
+            {profile.map(i => (
```
Comment (yehee) [inline client/src/pages/ContactPage.js]: Either give it a more meaningful name instead of `i` or destructure those values directly:
```suggestion
            {profile.map(({ name, github, linkedin, src }) => (
```
It's also missing `key` attribute

**Review by yehee — CHANGES_REQUESTED** [review]

**Review by yehee — APPROVED** [review]
